lib/middlewares.js

```diff
@@ -0,0 +1,19 @@
+
+const validateCreateRoomParams = (req, res, next) => {
+  const { eventNotificationUri, maxPeersRoom } = req.body;
+
+  // Validate that eventNotificationUri is a string
+  if (!eventNotificationUri || typeof eventNotificationUri !== 'string') {
+    return res.status(400).json({ error: 'Invalid or missing eventNotificationUri' });
+  }
+
+  // Validate that maxPeersCount is a number
+  if (!maxPeersRoom || typeof maxPeersRoom !== 'number') {
```

Comment:
I think this can be optional. In such case, no limit would be imposed

Reply:
Resolved: I added an additional validation for a number less than 2.